### PR TITLE
models: latest release for unbound instance fix + release: v1.0.0a4

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: invenio_github/models.py [hunk 1]
```diff
@@ -238,6 +238,9 @@ def enabled(self):
 
     def latest_release(self, status=None):
         """Chronologically latest published release of the repository."""
+        # Bail out fast if object not in DB session.
+        if self not in db.session:
```
Comment (jirikuncar) [inline invenio_github/models.py]: http://docs.sqlalchemy.org/en/latest/orm/session_basics.html#how-can-i-get-the-session-for-a-certain-object

Comment (lnielsen) [inline invenio_github/models.py]: It's due to [this](https://github.com/inveniosoftware/invenio-github/blob/master/invenio_github/views/github.py#L160-L161), which creates an instance which should go in the session.

http://docs.sqlalchemy.org/en/latest/orm/session_state_management.html#session-attributes

Comment (jirikuncar) [inline invenio_github/models.py]: If we do a session detection then we should do it correctly with the `inspect` method.

Comment (lnielsen) [inline invenio_github/models.py]: And why is ``obj not in db.session`` not correct as describe in above link?

**Comment by lnielsen** [comment]
We need this fix urgently so I have to merge now, I have however opened #19.

File: invenio_github/models.py (hunk 1 above)
Comment (jirikuncar) [inline invenio_github/models.py]: because the object can be in other session ...

Comment (lnielsen) [inline invenio_github/models.py]: but here we are just interested to know if it's actually in the current session

Comment (jirikuncar) [inline invenio_github/models.py]: The object can be binded to a different session and then you can not get latest release.

**Pull request merged**